src/server/api/lib/nexmo.js

```diff
@@ -45,8 +44,12 @@ async function convertMessagePartsToMessage(messageParts) {
 
 async function findNewCell() {
   if (!nexmo) {
+    const num = "1234"
+      .split("")
+      .map(() => parseInt(Math.random() * 10))
+      .join("");
     return {
-      numbers: [{ msisdn: getFormattedPhoneNumber(faker.phone.phoneNumber()) }]
+      numbers: [{ msisdn: getFormattedPhoneNumber(`+1212555${num}`) }]
```

Comment:
The majority of this number isn't randomized anymore, what is `nexmo.js` used for? Will that be a problem at all?